### DISCUSSION TITLE
Looking for maintainers ⚡

### DISCUSSION THREAD

**Discussion by saltyshiomix** [saltyshiomix/nextron#244] [General]
[Want to be a collaborator? #25](https://github.com/saltyshiomix/nextron/issues/25)

Feel free to comment and raise your hands for nextron 👍

### What I want to do with you...

- Solve the issues and discussions
- Review PRs
- Merge PRs
- Upsert examples for reference [vercel/next.js examples](https://github.com/vercel/next.js/tree/canary/examples)
- Maintenance package dependencies
- or any other what we thought as good

### I want to do myself for a while...

- npm publish

**Comment by Psycokwet** [comment]
Hello ! :)
I want to try and be a maintainer. I'm not sure I'm yet skilled enough to do proper evolutions, but I can still try my best :) I really like Nextron. (Thank you again for that, by the way !)

**Reply by saltyshiomix** [reply]
Thank you, @Psycokwet ! (Sorry for late reply)

You are already this repo's collaborators, so all reviews and merges are welcome :)

Feel free to mention me via GitHub, [Twitter DM](https://twitter.com/saltyshiomix) and any other way.

**Comment by m5x5** [comment]
Hello, I'd love to help improve the project as well! I've cloned the project once already to implement changes I'd like to have, and I'm sure I could fix a couple of issues sometimes.

**Reply by saltyshiomix** [reply]
Thank you for trying to improve nextron, @m5x5 !

I just added you as a collaborator 👍 
Please see https://github.com/saltyshiomix/nextron/invitations

**Reply by m5x5** [reply]
Thank you! I'll do my best to improve nextron 😁

**Comment by saltyshiomix** [comment]
I think **we, every collaborators** can do...

- solve the issues
- review PRs, merge PRs
- upsert examples for reference [vercel/next.js examples](https://github.com/vercel/next.js/tree/canary/examples)
- or any other what we thought as good

**Reply by Psycokwet** [reply]
Great ! I wasn't sure if my possibles upgrades would be appreciated... I will try to propose them as soon as I can :)

**Reply by saltyshiomix** [reply]
@Psycokwet 
We can fix them before releases, so don't worry your upgrades would be correct or not :)

**Comment by andirsun** [comment]
@saltyshiomix at [Streamline](https://github.com/webalys-hq) we are interested in maintain this package as we are using it for our desktop app. Please review our first PR and let us be one of the mantainers, btw awesome work! **https://github.com/saltyshiomix/nextron/pull/262**

**Reply by saltyshiomix** [reply]
@andirsun 
Sorry for late reply.
I just added you as a collaborator: https://github.com/saltyshiomix/nextron/invitations

**Reply by andirsun** [reply]
Thanks dude

**Reply by saltyshiomix** [reply]
@andirsun 

Sorry for late updating README...

I just added you as "Maintainers" in the README :)

https://github.com/saltyshiomix/nextron#maintainers-

**Comment by knownasnaffy** [comment]
Hello, I'm a bit new to nextron, but I have worked with nextjs and electron. I want to update the documentation of this package as searching through the README file is a bit annoying task.

I hope I can do good work. 😀

**Reply by knownasnaffy** [reply]
@saltyshiomix No replies 🙄

**Reply by andirsun** [reply]
Please, before considering your application first upload your first PR.

**Reply by knownasnaffy** [reply]
Ok thanks 😅

**Reply by andirsun** [reply]
Thanks to you!

**Comment by bm777** [comment]
I think the best way to contribute is to learn through issues from others. I was at the fastAPI and flask creators meetup, and they said from all the collaborators in their repo, few of them are really contributing (max 5 folks), they pointed out that **replying to the issue** is very important in the contribution field.

I would like to help in 
- replying to issues (eventually solving them)
- support for Linux/Mac environment

----

Hoping to help @saltyshiomix ✌️

**Reply by saltyshiomix** [reply]
@bm777 

Thank you for your advice!
I really appreciate your help and your creating [the issue](https://github.com/saltyshiomix/nextron/issues/405) :)

**Comment by pixelass** [comment]
So we started using nextron for [Captain](https://github.com/blib-la/captain) and sadly run into major issues on a daily basis (biggest issue for us is this: https://github.com/saltyshiomix/nextron/issues/333). 

If it needs fixing... fix it. Wondering how I can chime in to help.

I'm a long time Next.js developer, pretty new to Electron (only built one app ~6 years ago).

I'm really good at MUI and general frontend logic/architecture and docs:

- [example 1](https://github.com/blib-la/blibla-template)
- [example 2](https://github.com/blib-la/hyv)

Just tell me where the shovels are... :)

**Reply by saltyshiomix** [reply]
Sorry for late reply and inconvenience.

Ok, I just assigned you as a collaborator :)
https://github.com/saltyshiomix/nextron/invitations

And, I'll support you as best as I can.
Please contact me via GitHub Issues **and** twitter DM: https://twitter.com/saltyshiomix

**Comment by sulekaustubh** [comment]
Hello. I genuinely want to know if and when in future you will add support for Next.js v14 for nextron.

**Reply by saltyshiomix** [reply]
@sulekaustubh 
Sorry for inconvenience.
Nextron v9 already supports Next.js v14 :)

Please update renderer/next.config.js as belows:

```js
module.exports = {
  output: 'export',
  // we want to change distDir to "app" so as nextron can build the app in production mode!
  distDir: process.env.NODE_ENV === 'production' ? '../app' : '.next',
  trailingSlash: true,
  images: {
    unoptimized: true,
  },
}
```

**Reply by sulekaustubh** [reply]
Thank you so much for bringing Next.js to desktop development! 😄 
I realized it later that it already had Next.js v14 running. Sorry. @saltyshiomix

**Reply by saltyshiomix** [reply]
@sulekaustubh 
No problem!! Feel free to contact me if any troubles :)

**Reply by makhdum530** [reply]
[nextron] We must export static files so as Electron can handle them. Please set next.config.js#output to "export".
